CaT/src/Filter/Navigator.php

```diff
@@ -107,6 +107,6 @@ protected function getItemByPath($path, $tmp) {
 	}
 
 	public function path() {
-		return implode("_",$this->path);
+		return is_array($this->path) ? implode("_",$this->path) : null;
```

Comment:
Das ist in so in Ordnung. Wenn hier NULL zurück kommt, wird $this->path im Anschluss initialisiert.
Man kann $this->path nicht vorher initialisieren, da man sonst nie den ersten Filter bekommen würde, sondern immer erst den Zweiten.